### PR TITLE
IC-1883: Display additional details on service user page

### PR DESCRIPTION
**Pull request by andrewburrowsmoj**
(subject to change based on feedback from Macin/Debbie)

## What does this pull request do?

Exposes email, phone number and address for service user on Service User Details page and Check Your Answers page.

## What is the intent behind these changes?

Ensures that PP user can verify if the email, phone number and address is correct from Delius.

## Notes

Still need confirmation:
Assuming that we should not store these values. Please correct me if that's a wrong assumption.
Changes are visible on the following three pages:

1. Referral Details
image

2. Service User Details (as part of Make a referral)
image

3. Check Your Answers (as part of Make a referral)
image

### CODE REVIEW

File: server/routes/referrals/serviceUserDetailsPresenter.ts [hunk 1]
```diff
@@ -12,14 +12,25 @@ export default class ServiceUserDetailsPresenter {
...
   readonly title = `${this.serviceUser.firstName || 'Service user'}'s information`
 
-  get summary(): SummaryListItem[] {
-    // required to force type erasure of type null[] from list of (string[] | null[])
-    function notEmpty<TValue>(value: TValue | null | undefined): value is TValue {
-      return value !== null && value !== undefined
-    }
-    const phoneNumbers: string[] | undefined = this.deliusServiceUserDetails.contactDetails.phoneNumbers
+  // required to force type erasure of type null[] from list of (string[] | null[])
+  private notEmpty<TValue>(value: TValue | null | undefined): value is TValue {
```
Comment (Gweaton) [inline server/routes/referrals/serviceUserDetailsPresenter.ts:16]: I've found this function a bit hard to parse, but it's probably because I'm not good enough at Typescript - I don't suppose there's some way we can make it more readable, is there? I think it's the name `TValue` that's confusing me.

**Review by Gweaton — APPROVED** [review]
Just a couple of minor comments, apologies for the delay in reviewing.

File: server/routes/referrals/serviceUserDetailsPresenter.ts (hunk 1 above)
Comment (andrewburrowsmoj) [reply server/routes/referrals/serviceUserDetailsPresenter.ts:16]: TValue is for the generic Type T, I'll change it to String since we're dealing with Strings

Comment (Gweaton) [reply server/routes/referrals/serviceUserDetailsPresenter.ts:16]: Thanks!

Comment (andrewburrowsmoj) [reply server/routes/referrals/serviceUserDetailsPresenter.ts]: done

File: server/routes/referrals/checkAnswersPresenter.test.ts [hunk 2]
```diff
@@ -22,23 +23,7 @@ describe(CheckAnswersPresenter, () => {
...
-    title: null,
-    firstName: null,
-    surname: null,
-    dateOfBirth: null,
-    gender: null,
-    contactDetails: {
-      emailAddresses: ['alex.river@example.com', 'alex61@example.com'],
-    },
-  }
+  const deliusServiceUser = deliusServiceUserFactory.build()
```
[the remark this answers is not in the document]
Comment (andrewburrowsmoj) [reply server/routes/referrals/checkAnswersPresenter.test.ts]: thanks, I've changed to this
